operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,198 @@
+  o- loopback ......................................................................................................... [Targets: 0]
+  o- vhost ............................................................................................................ [Targets: 0]
+  o- xen-pvscsi ....................................................................................................... [Targets: 0]
+```
+In above 'targetcli ls' command output, we can see that four fileio backing store is created for four rootfs images and four iSCSI LUNs created which has the 'rootfs' id being mapped and ready for rootfs image projection. 
+
+Sample initiator node snippet after image projection:
+```bash
+nid000002:~ # multipath -ll
+f1d6c8fe-32e2-420a-a051-377e34a8bd8a_rootfs (36001405ad7d9e9736a1b8a13f16fad7a) dm-9 LIO-ORG,ad7d9e9736a1b8a
+size=3.6G features='0' hwhandler='1 alua' wp=ro
+`-+- policy='round-robin 0' prio=50 status=active
+  |- 10:0:0:10 sdaj 66:48  active ready running
+  |- 8:0:0:10  sdd  8:48   active ready running
+  |- 9:0:0:2   sdy  65:128 active ready running
+  `- 11:0:0:2  sdax 67:16  active ready running
+```
+
+# Steps to achieve SBPS
+
+## Node Personalisation (Worker node)
+As part of iSCSI SBPS solution, we need to first setup/ configure worker nodes as iSCSI targets (servers) with necessary provisioning, configuration and enablement of required components. The required RPMs for targetcli command / LIO are part of NCN node image in CSM 1.6 and SBPS Marshal agent gets installed during node personalisation via CFS Ansible play books.
+
+See https://github.com/Cray-HPE/docs-csm/blob/CASMPET-6934-SBPS-Node-Personalization/operations/configuration_management/iSCSI_SBPS_Node_Customization.md for more info.
+
+## Steps to configure and run GOSS tests 
+In order to verify the readiness of the iSCSI targets before triggering the boot of compute nodes/ UAN's, we need to run GOSS tests as sanity checks on iSCSI targets.
+
+See https://github.com/Cray-HPE/sbps-marshal/blob/CASMPET-7214-goss-sbps-doc/GOSS_tests_for_sbps.md for more info.
+
+## IMS images tagging
+To trigger the boot of compute node/ UANs, the images (rootfs/ PE ) are tagged to determine which rootfs image to be projected for compute/ UANs as there will be many rootfs/ PE images. SBPS Marshal agent uses key/value pair of sbps-project/true to identify the image(s) tagged. IMS images (rootfs) tagging is supported from IMS initially which requires to be done manually using craycli. 
+
+Command to tag an IMS image using craycli 
+```bash
+cray ims images update < Image id > --metadata-operation set --metadata-key < key > --metadata-value < value >
+```
+e.g.
+```bash
+cray ims images update bbe0e9eb-fa8f-4896-9f54-95dbd26de9bb --metadata-operation set --metadata-key sbps-project --metadata-value true
+```
+Command to describe the IMS image metadata:
+```bash
+cray ims images describe < Image id > --format json

```

Comment:
Same thing, remove the `bash`, and change `< Image id >` to `<image_id>`

Reply:
done